### PR TITLE
code problem in ssh.py

### PR DESCRIPTION
**Pull request by Junefen**
It needs double (( )) in```transport = paramiko.Transport((self.host, self.port))```

### CODE REVIEW

**Review by eko999 — COMMENTED** [review]
If that, not only `Transport`, we should define the self.port as `int`, could you try:
```
 self.port = int(port)
```

**Comment by Junefen** [comment]
> If that, not only `Transport`, we should define the self.port as `int`, could you try:
> 
> ```
>  self.port = int(port)
> ```

Yes the port should be int and it's better to define it here.

**Review by hkx303 — APPROVED** [review]
ACK

File: virtwho/ssh.py [hunk 1]
```diff
@@ -69,15 +69,15 @@ def rsa_connect(self):
 
     def pwd_transfer(self):
         """Sftp download/upload execution connection by password"""
-        transport = paramiko.Transport(self.host, self.port)
+        transport = paramiko.Transport((self.host, self.port))
```
Comment (eko999) [inline virtwho/ssh.py:72]: I think we don't need this change now, `int(port)` is everything

File: virtwho/ssh.py [hunk 2]
```diff
@@ -69,15 +69,15 @@ def rsa_connect(self):
...
         transport.connect(username=self.user, password=self.pwd)
         sftp = paramiko.SFTPClient.from_transport(transport)
         return sftp, transport
 
     def rsa_transfer(self):
         """Sftp download/upload execution connection by key file"""
         pkey = paramiko.RSAKey.from_private_key_file(self.rsa)
-        transport = paramiko.Transport(self.host, self.port)
+        transport = paramiko.Transport((self.host, self.port))
```
Comment (eko999) [inline virtwho/ssh.py:80]: same as above

File: virtwho/ssh.py (hunk 1 above)
Comment (eko999) [reply virtwho/ssh.py:72]: Yes, you are right, I tested this, the (( )) is needed ...

**Review by eko999 — APPROVED** [review]

**Pull request merged**